session-1/session2.html

```diff
@@ -0,0 +1,143 @@
+                            <a class="nav-link active" href="#">Skills</a>
+                        </li>
+                        <li class="nav-item me-3">
+                            <a class="nav-link active" href="#">Portofolio</a>
+                        </li>
+                        <li class="nav-item me-3">
+                            <a href="#" class="nav-link active btn btn-success rounded" style="color: #fff;">Contact</a>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </nav>
+    </header>
+    <!-- Content -->
+    <main class="p-5 py-5 mt-5">
+        <h1 class="text-center text-decoration-underline mt-3 fw-bold">Experience</h1>
+        <div class="container">
+            <div class="row">
+                <div class="col-12 col-md-6 col-sm-6 py-3 align-self-center center">
+                    <h2 class="display-5 fs-1 fw-bold">What's my experience?</h2>
+                    <p class="fs-3">Describe your experience in 20 words or more. Such as Insternship experience, Bootcamp, Student Exchange and so on.</p>
+                </div>
+                <div class="col-12 col-md-6 col-sm-6 py-3 card border-0">
+                    <div class="row">
+                        <div class="col-12 col-md-6 col-sm-6 card border-0 rounded shadow-sm w-100 box-card our-services mb-3 mt-3">
+                            <div class="card-title">
+                                <h3 class="fw-bold">UI/UX Designer</h3>
+                                <p>PT.Persentologic Indonesia <span>· Internship</span></p>
+                                <p class="text-muted fs-5">January 2022 - Present</p>
+                                <p class="fs-5 lh-sm">User Experience researcher and create user interfaces for PT.Persentologic Indonesia landing pages.
+                                </p>
+                            </div>
+                        </div>
+                        <div class="col-12 col-md-6 col-sm-6 card border-0 rounded shadow-sm w-100 box-card our-services mb-3 mt-3">
+                            <div class="card-title">
+                                <h3 class="fw-bold">Graphic Designer</h3>
+                                <p>Upwork <span>· Freelance</span></p>
+                                <p class="text-muted">January 2021 - Present</p>
+                                <ul class="fs-5 lh-sm">
+                                    <li>Making Logo for Company, Group such as E- Sports, Bussiness, and Personal user.</li>
+                                    <li>Making Avatar or Vector Illustration.</li>
+                                    <li>Designing Pamphlet, Banner, Info Graphics, etc.</li>
+                                </ul>
+                            </div>
+                        </div>
+                        <div class="col-12 col-md-6 col-sm-6 card border-0 shadow-sm w-100 box-card our-services mb-3 mt-3">
```

Comment:
don't mix column with card class, add card-body class. Read again the documentation [card](https://getbootstrap.com/docs/5.1/components/card/)